GameServerLib/ProtectionManager.cs

```diff
@@ -56,7 +56,7 @@ public void RemoveProtection(IAttackableUnit element)
 
         public bool IsProtected(IAttackableUnit element)
         {
-            return _protectedElements.Contains(element);
+            return _hasProtectionElements.Contains(element);
```

Comment:
The previous function was technically correct. Separate this into another `IsProtectionActive(IAttackableUnit element)` function instead.

Reply:
I have not yet added a new function to the ApiFunctionManager or renamed an existing one in it, because its description, which I did not touch, corresponds to the purpose of IsProtectionActive